src/EditorFeatures/CSharpTest/EditorConfigSettings/DataProvider/DataProviderTests.cs

```diff
@@ -130,7 +130,7 @@ public void TestGettingFormattingSettingProviderLanguageServiceAsync()
             var model = new TestViewModel();
             settingsProvider.RegisterViewModel(model);
             var dataSnapShot = settingsProvider.GetCurrentDataSnapshot();
-            Assert.Equal(47, dataSnapShot.Length);
+            Assert.Equal(48, dataSnapShot.Length);
```

Comment:
![image](https://user-images.githubusercontent.com/4404199/116919542-17b98380-ac5a-11eb-8951-4d47ec695835.png)

This error was in the build, so I simply incremented the expected value to 48 since a new setting was added. Is this correct?